static/sass/_snapcraft_details_heading.scss

```diff
@@ -46,7 +46,7 @@
   .p-snap-install {
 
     &__versions {
-      margin-top: 73px; // align with install instructions
+      margin-top: 68px; // align with install instructions
```

Comment:
@therealjuan Is this 68px used to align buttons with something? Snap name baseline I guess? - then we should update the comment in code, as it still says about install instructions.

Reply:
magic number alert :)